code/game/turfs/turf.dm

```diff
@@ -132,6 +145,19 @@ GLOBAL_LIST_EMPTY(created_baseturf_lists)
 		update_air_ref(-1)
 		__auxtools_update_turf_temp_info(isspaceturf(get_z_base_turf()))
 
+	if(traction)
+		make_traction()
+
+	//Handle turf texture
+	if(get_turf_texture())
+		add_turf_texture(get_turf_texture())
+
+	//Handle icon border
+	if(icon_x_offset || icon_y_offset)
+		var/matrix/M = new
+		M.Translate(-icon_x_offset, -icon_y_offset)
```

Comment:
We have a DEFINE for this now used by volcanic turfs. Just manually specify the transform statically with that rather than adding this check to every turf init